Python/bytecodes.c

```diff
@@ -0,0 +1,3795 @@
+#include "Python.h"
+
+#include "opcode.h"
+#include "pycore_atomic.h"
+#include "pycore_frame.h"
+
+void _PyFloat_ExactDealloc(PyObject *);
+void _PyUnicode_ExactDealloc(PyObject *);
+
+#define SET_TOP(v)        (stack_pointer[-1] = (v))
+#define GETLOCAL(i)     (frame->localsplus[i])
+
+#define inst(name, stack_effect) case name:
+#define family(name) static int family_##name
+
+static void
+dummy_func(
+    PyThreadState *tstate,
+    _PyInterpreterFrame *frame,
+    unsigned char opcode,
+    unsigned int oparg,
+    _Py_atomic_int * const eval_breaker,
+    _PyCFrame cframe,
+    PyObject *names,
+    PyObject *consts,
+    _Py_CODEUNIT *next_instr,
+    PyObject **stack_pointer
+)
+{
+    switch (opcode) {
```

Comment:
The indentation is not the issue. If each instruction lived in its own function, references to variables like `frame` or `eval_breaker` would all get red wiggly lines in VS Code. Every single variable I listed as a parameter to `dummy_func()` above is needed to avoid wiggly lines. There are also some labels at the bottom of the file that are needed as the target of `goto` labels.